src/test/java/io/shapelets/khiva/LibraryTest.java

```diff
@@ -67,7 +86,84 @@ public void testGetDevice() {
     }
 
     @Test
-    public void testGetKhivaVersion() {
-        Assert.assertEquals(Library.getKhivaVersion(), "0.1.0");
+    public void testGetKhivaVersion() { Assert.assertEquals(Library.getKhivaVersion(), getKhivaVersionFromGithub()); }
+
+    private String getKhivaVersionFromGithub() {
+        String response = getTagsFromGitHub();
+        Gson gson = new GsonBuilder().create();
+        GithubTag[] tagsArray = gson.fromJson(response, GithubTag[].class);
+        String version = tagsArray[tagsArray.length-1].getName();
+        version = version.replace("v", "");
+        version = version.replace("-RC", "");
+
+        return version;
+    }
+
+    private String getTagsFromGitHub() {
+        String response = "";
+        CloseableHttpClient httpClient = HttpClientBuilder.create().build();
+        try {
+            HttpGet httpGetRequest = new HttpGet("https://api.github.com/repos/shapelets/khiva/tags");
+            HttpResponse httpResponse = httpClient.execute(httpGetRequest);
+            HttpEntity entity = httpResponse.getEntity();
+
+            byte[] buffer = new byte[1024];
+            if (entity != null) {
+                InputStream inputStream = entity.getContent();
+                try {
+                    int bytesRead = 0;
+                    BufferedInputStream bis = new BufferedInputStream(inputStream);
+                    while ((bytesRead = bis.read(buffer)) != -1) {
+                        response = response + new String(buffer, 0, bytesRead);
+                    }
+                } catch (Exception e) {
+                    e.printStackTrace();
+                } finally {
+                    try { inputStream.close(); } catch (Exception ignore) {}
+                }
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        return response;
+    }
+
+    public static class GithubTag {
```

Comment:
can this be private? is only used by the getKhivaVersionFromGithub() method, which is private